poseidon/src/parameters.rs

```diff
@@ -1,2 +1,81 @@
-use ark_ff::vec;
-include!(concat!(env!("OUT_DIR"), "/parameters.rs"));
+use ark_ff::PrimeField;
```

Comment:
we may add a header with a warning about not changing this file

Reply:
Yeah, I'm about to do that.